<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addmember</class>
 <widget class="QDialog" name="addmember">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(255, 255, 222);	
}

QComboBox{
	border: 1px solid #aaaa7f;
}
QLineEdit{
	border: 1px solid #aaaa7f;
}
QSpinBox{
	border: 1px solid #aaaa7f;
}
QTextEdit{
	border: 1px solid #aaaa7f;
}
QDatetEdit{
	border: 1px solid #aaaa7f;
}
QGroupBox{
	border: 1px solid #aaaa7f;
}
QDateTimeEdit{
	border: 1px solid #aaaa7f;
}


QLabel#label_23,#label_25,#label_26,#label_53,#label_54,#label_55,#label_56{
	color: rgb(255, 0, 0);
}



QPushButton{background-color:rgb(255, 255 , 178);

	color: black;  
	border-radius: 10px; 
	border: 2px groove gray; 
	min-height:20;
 	border-style: outset;
}

QPushButton:hover{background-color: rgb(255, 219, 92);
color: black ;
}

QPushButton:pressed{background-color:rgb(255, 175, 68);

border-style: inset; }</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>会员信息</string>
   </property>
   <widget class="QComboBox" name="comboBox_memberType">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>一级会员</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>二级会员</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>三级会员</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_confirmPwd">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>572</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>备注：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_useMeberPrice">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>使用会员价</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_useDiscount">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>使用折扣率</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_birthWarning">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>当天提醒</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>提前一天</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>提前两天</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>提前三天</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>提前一周</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_memberPhone">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>100</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_userCredit">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>152</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>178</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_qq">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>204</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_address">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>230</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>初始余额：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_initialBalance">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>累积积分：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>账户余额：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>590</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>正常</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>590</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>停用</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>590</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>状态：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>状态：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_state">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>正常</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>停用</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>备注：</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEditBirthday">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy/M/d </string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>会员类型：</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit_begin">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>124</width>
      <height>20</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2015</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="displayFormat">
     <string>yyyy/M/d </string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>开卡日期：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>卡号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_memberId">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>48</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_memberName">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>74</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电话：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>生日：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>生日提醒：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>152</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>证件号码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>邮箱：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>206</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>QQ：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>地址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>256</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>会员密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pwd">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>256</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>确认密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>初始积分：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_initialIntegral">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>289</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>会员折扣：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_memberDiscount">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>322</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>99%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>98%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>97%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>96%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>95%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>94%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>93%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>92%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>91%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>90%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>89%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>88%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>87%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>86%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>85%</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_integral">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>292</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_balance">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>352</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>480</y>
     <width>158</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="saveMemberInfo">
      <property name="text">
       <string>保存(S)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>取消(C)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
